src/templates/fluent.conf

```diff
@@ -4,6 +4,18 @@
   bind 0.0.0.0
 </source>
 
+<filter **>
+  @type record_transformer
+  enable_ruby
+  <record>
+    message "${record['log'].split(' ')[0]}T${record['log'].split(' ')[1]} ${record['log'].split(' ')[2..-1].join(' ')}, ${record['container_name'].split('_')[1..-1].join('_')}"
```

Comment:
IMO container name should go first kinda like bitcoin test combined logs

Reply:
https://gist.github.com/pinheadmz/477ea23854bf510d2acad929bf15a20a

Reply:
Done